app/.eslintrc.json

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "next/core-web-vitals"
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "nava",
+    "plugin:@typescript-eslint/recommended",
+    "prettier" 
+  ],
+  "rules": {
+    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/no-explicit-any": "error"
```

Comment:
Curious: Why these particular rules?

Reply:
I felt like these were two good rules to start with -- I talked about why `any` types are strongly advice against in my ts style guide. (Its usage defeats the purpose of typescript). RE: no-unused vars -- it just checks for any unused variables. I noticed that pfml has _a lot_ of rules, so I wanted to ask Sawyer about this when he gets back from PTO.

Reply:
Those sound great and totally make sense to me. Let's document them in the README.